doc/input-output-reference/src/overview/group-thermal-zone-description-geometry.tex

```diff
@@ -2725,6 +2726,10 @@ \subsubsection{Surface Inside Face Absorbed Shortwave Radiation Rate {[}W{]}}\la
 
 As of Version 2.1, the previous variable plus absorbed shortwave radiation from uniformly distributed initially-reflected diffuse solar, reflected beam solar, and shortwave radiation from lights. This sum is the power of all sources of solar and visible radiation absorbed by the surface at the inside face.
 
+\subsubsection{Surface Construction Index {[]}}\label{surface-construction-index}
+
+The variable reports an index that points to specific Construction. The variable is an additional output variable when Output:Diagnostics, DisplayAdvancedReportVariables; is given in an input file.
+ 
```

Comment:
So how is the user able to match this integer with a surface construction? Could you provide guidance on how this is done? See eio report using "Re;portConstruction" (or whatever is needed to get those reports)